lib/node_modules/@stdlib/repl/lib/editor_actions.js

```diff
@@ -523,21 +523,18 @@ setNonEnumerableReadOnly( EditorActions.prototype, 'clearScreen', function clear
 * @returns {boolean} boolean indicating whether an editor action was triggered
 */
 setNonEnumerableReadOnly( EditorActions.prototype, 'beforeKeypress', function beforeKeypress( data, key ) {
-	var actionKeys;
 	var inputKeys;
+	var action;
 	var i;
 
 	if ( !key ) {
 		return false;
 	}
 	inputKeys = parseKey( key );
-	for ( i = 0; i < EDITOR_ACTIONS.length; i++ ) {
-		actionKeys = this._keybindings[ EDITOR_ACTIONS[ i ] ];
-		if ( !actionKeys ) {
-			continue; // no keybindings configured for the action
-		}
-		if ( matchKeybindings( inputKeys, actionKeys ) ) {
-			this[ EDITOR_ACTIONS[ i ] ]();
+	for ( i = 0; i < inputKeys.length; i++ ) {
+		action = this._keybindings.getAction( inputKeys[ i ] );
+		if ( action && contains( EDITOR_ACTIONS, action ) ) {
```

Comment:
@Snehil-Shah You can use the `array/base/assert/contains` factory to create a faster `isAction` function.